ci/webhook-receiver.py

```diff
@@ -196,13 +204,30 @@ def init_ci_pipeline():
     out, err = proc2.communicate()
 
     if (out):
-        print("Can't run CI, another CI run in progress")
-        log_access_granted(request_ctx, "CI busy, posting busy msg")
-        os.system("./ci_busy.sh config {} \"{}\" \"{}\" \"Another CI run in progress, please try again in 15 minutes\""
-                  .format(request_ctx['id'], request_ctx['repo'], request_ctx['body']))
+        print("Can't run CI, another CI run in progress, placing in the queue")
+        log_access_granted(request_ctx, "CI busy, placing request in queue")
+        duplicate_req = False
+        busy_msg = "Duplicate request already waiting, ignoring message"
+        with queue_lock:
+            # make sure we don't bypass thread safety
+            for req in ci_list:
+                if req['id'] == request_ctx['id'] and req['repo'] == request_ctx['repo']:
+                    # make sure this is the same PR
```

Comment:
same, move comment above, usual structure for comments is this
```
// Check if ctulhu is summoned
if (ктулъхе == тута)
```

Reply:
lmao at the example code